extensions/elasticsearch/.lando.yml

```diff
@@ -0,0 +1,32 @@
+services:
+  elasticsearch:
+    type: compose
+    services:
+      image: "docker.elastic.co/elasticsearch/elasticsearch:7.17.0"
```

Comment:
Maybe we should set version to 8.11 already?

Reply:
Yes, I agree. We should be keeping up to date with EOL dates. I moved these to _7 extensions: https://github.com/wunderio/lando-drupal/pull/29/commits/60575e2d4aae893656dd681a5fcbe0b899c701e4

![image](https://github.com/wunderio/lando-drupal/assets/492375/fe6528dc-12c4-4fee-997a-0fe5d5b0e46f)

The one without version is default.. or rolling version. If 8 would get EOL, we would update it. But I wonder if anybody dares to use it and we should have versions for all so elasticsearch would become elasticsearch_8

@deiviic

Reply:
It would also maybe be more intuitive for somebody who is looking at this for the first time.

Reply:
@deiviic I need to merge this branch and create a release but seems 8.x didn't start to work. I've created new issue dealing with that plus naming question https://github.com/wunderio/lando-drupal/issues/32